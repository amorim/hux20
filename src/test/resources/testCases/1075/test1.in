7 7
1 2 10
1 3 7
2 4 1
3 4 15
5 4 2
5 6 5
6 7 8