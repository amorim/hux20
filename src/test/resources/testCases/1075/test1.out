It's possible with distance 11